Ukoly/DU_4/Caesar.py

```diff
@@ -0,0 +1,52 @@
+# vytvoření seznamu čísel, která jsou kódy pro písmena abecedy
+rank = list(range(65, 123))
+czech = ['ě', 'š', 'č', 'ř', 'ž', 'ý', 'á', 'í', 'é', 'ň', 'ď', 'ů', 'ú']
```

Comment:
Taký menší hint... ak to chceš spraviť len na znaky anglickej abecedy, tak si pozri ako sú tieto znaky rozložené v ASCII tabulke...